ansible/roles/vendor/requirements.yml

```diff
@@ -27,3 +27,5 @@
 - src: https://github.com/GSA/rvm1-ansible
   version: v1.3.8-fix
   name: gsa.rvm1-ruby
+
+- src: https://github.com/GSA/datagov-deploy-redis
```

Comment:
in order to use the role in a playbook we need to add a `name` parameter for the remote git repo to reference it
```
- src: https://github.com/GSA/datagov-deploy-redis
   name: gsa.datagov-deploy-redis
```

Otherwise looks good